SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestReplicationConditions/TestReplicationConditionsActor.cpp

```diff
@@ -190,6 +190,11 @@ void ATestReplicationConditionsActor_AutonomousOnly::SpawnDynamicComponents()
 		TEXT("DynamicTestReplicationConditionsComponent_AutonomousOnly"));
 }
 
+bool UTestReplicationConditionsPrimitiveComponent::IsSimulatingPhysics(FName BoneName /*= NAME_None*/) const
+{
+	return true;
```

Comment:
Shouldn't this return what we set in `RootPrimComp->SetSimulatePhysics(bEnabled);` ?